linux_os/guide/system/network/network-kernel/network_host_and_router_parameters/sysctl_net_ipv4_tcp_syncookies/rule.yml

```
@@ -53,3 +53,8 @@ template:
     vars:
         sysctlvar: net.ipv4.tcp_syncookies
         datatype: int
+
+fix: |-
+   Configure {{{ full_name }}} to use TCP Syncookies on IPv4 interfaces.
```

```suggestion
   Configure {{{ full_name }}} to use TCP SYN cookies.
```
Also applies on IPv6 as well. See https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt. Search for IPv6.